CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
     * For more information, visit https://docs.labelbox.com/reference/migrating-to-workflows#upcoming-changes
 * Updated `project.export_labels` to support filtering by start/end time formats "YYYY-MM-DD" and "YYYY-MM-DD hh:mm:ss"
 * Add deprecation warning for missing `media_type` in `create_project` in `Client`.
+* Update QueueMode enum to support new value for QueueMode.Batch = `BATCHES`.
```

Comment:
may want to rebase and move this line under the "in progress" section when a new version has been released